src/app/Components/Home/ArtistRails/ArtistCard.tsx

```diff
@@ -16,7 +16,7 @@ interface ArtistCardProps {
   onPress?: () => void
 }
 
-const IMAGE_MAX_HEIGHT = 180
+export const IMAGE_MAX_HEIGHT = 180
```

Comment:
Might be good to choose a name related to the artist rail if we export it. Could be something like:

```suggestion
export const ARTIST_RAIL_MAX_IMAGE_SIZE = 180
```

Reply:
I feel like it's kind of clear here since it's already insde the `ArtistCard` component. I will update it in the import thought to be something like
```typescript
import { IMAGE_MAX_HEIGHT as ARTIST_RAIL_MAX_IMAGE_SIZE } from ...
```

Reply:
I guess, it's ok like it is 👍 I'm just concerned about exporting things with a generic name because it could confuse editor's auto-import feature...